src/collectors/colmenaConfigurations.nix

```diff
@@ -1,12 +1,14 @@
 {
   inputs,
   nixpkgs,
-  cellBlock,
+  root,
 }: renamer: let
+  cellBlock = "colmenaConfigurations";
```

Comment:
Note to self: figure out a way to avoid the stringly typed signal. Would be nice if we could pass `cellBlocks` metadata instead and figure out the names from the matching block type.

But that'll be a design improvement and a breaking change, not a mere refacor as the scope of this PR.